docs/backup-restore.md

```diff
@@ -24,6 +24,34 @@ That means that you need to create a consistent snapshot by dumping all tables i
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
 
+## Cosmos DB
+
+You can create Cosmos DB account using `Continuous` backup policy to create continuous backups.
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on Cosmos DB is to implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
```

Comment:
Let's not create copy-paste redundant descriptions. 
If this needs to be explained several times, let's create a section for this and make other sections refer to this as necessary.

Reply:
Sorry for the convenience.

Document updated with a common section 01ae20a